vars/liskBuild.groovy

```diff
@@ -4,7 +4,7 @@ def call(Map params) {
 	def liskBranch = params.branch
 	def buildNetwork = params.network
 
-	def liskDir = "/tmp/lisk-${liskBranch}-${Math.abs(new Random().nextInt() % 1000) + 1}"
+	def liskDir = "lisk-${liskBranch}"
```

Comment:
The purpose of having that random integer was to have seperate repo for every build process. In normal jenkins files we can run the build process in parallel, and those can be done on same branches and same versions as well. So if you don't have seperate repo for each, the files will be overwritten.

Reply:
Every run of jenkins would run on a separate parent folder, as it's configured in:

     customWorkspace "workspace/${URLDecoder.decode(JOB_NAME)}/${BUILD_NUMBER}"

I don't see where they could overlap.

Reply:
Also, we clean the workspace at every run to prevent fulfilling the disk, with `cleanWs()`, which has no effect using `/tmp`

Reply:
@abelboldu You didn't understood. I am not saying every job, I am referring to parallel steps in same job. 

https://github.com/LiskHQ/lisk/blob/851a36c2e41fdcc2183cd536ae09d78488406fc1/Jenkinsfile.migration#L31-L33

Reply:
@abelboldu I agree that we can move it to workspace but we can't remove random postfix to the directory.

Reply:
Oh I see. you're calling it with two different versions for the migration use case. 
I don't think they would conflict anyway, because the version is included in the path, but let's keep the suffix to be safe.

Reply:
@abelboldu They definitely conflict, as we are using branch in directory name. And same branch can be used to build different lisk-core branches, which could have same version in their config.json file.